src/Sentry/Laravel/Features/Storage/Integration.php

```diff
@@ -48,10 +48,12 @@ function (Application $application, array $config) use ($filesystemManager): Fil
 
                     $diskResolver = (function (string $disk, array $config) {
                         // This is a "hack" to make sure that the original driver is resolved by the FilesystemManager
```

Comment:
This was originally added by @stayallive in https://github.com/getsentry/sentry-laravel/pull/726/commits/81c5692b8c65436ca0fdf1a6de8cf2305267bf85.